server.py

```diff
@@ -444,6 +445,6 @@ def do_not_track():
         context = ssl.SSLContext(ssl.PROTOCOL_TLSv1_2)
         context.load_cert_chain("local_data/cert.pem", "local_data/key.pem")
 
-        app.run(debug=True, host="0.0.0.0", port=7777, ssl_context=context)
+        app.run(host="0.0.0.0", port=7777, ssl_context=context)
```

Comment:
These lines are only executed in local environment so debug (gunicorn is used on production) should always be True I think.

Reply:
This is one of the step I want to introduce the debug flag to the flask `app` instance. Because from several previous PRs, you added the `flask profiler` dependency, so we can turn it on/off just by checking the debug flag

Reply:
I think it's more clear that each setting does only one thing and ideally, should be independent to the others. If someone wants to turn FLASK_PROFILER off, he can leave this section uncommented (it's uncommented by default).  "umbrella" setting like DEBUG tends to affect other settings, from my experience it usually causes confusion.

Reply:
You're right, I used it in my development workflow but didn't think too much on the downside. It is fixed in the last PR.